model/email.py

```diff
@@ -44,6 +45,12 @@ def has_credentials(self):
             return True
         return False
 
+    def filter_recipients(self, recipients:typing.Iterable[str]):
+        blacklist = self.raw.get('blacklist')
```

Comment:
nit: this `if` is not needed, if using the empty tuple as fallback value, like so:

```
blacklist = self.get.get('blacklist', ())
```